invenio_preservation_sync/models.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Preservation-Sync is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Models for Preservation Sync integration."""
+
+import uuid
+from enum import Enum
+
+from invenio_db import db
+from invenio_i18n import lazy_gettext as _
```

Comment:
If there are no translations in this module (or yet),  I would remove everything about i18n, including `babel.ini`, tests and CI.